main.cpp

```diff
@@ -131,27 +193,27 @@ int main()
     auto carRented = rentACar(6, 2); 
     
     //1)
-    
+    numberOfBirds(2, true);
     //2)
-    
+    frequencyRange(75.5f, 600.33f);
     //3)
-    
+    eatBeforeLunch(13.0f, true);
     //4)
-    
+    reverbTime(3.4, 2.1);
     //5)
-    
+    boilAnEgg(504.3f, true);
     //6)
-    
+    spaceOnTrain(40, 29);
     //7)
-    
+    bpm(true, 112.3f, 4);
     //8)
-    
+    waterPlants(false, 4);
     //9)
-    
+    modulation(5.32f, 0.3232f);
     //10)
+    myInitials(66);

```

Comment:
All of these functions return a value.  store the value in a local variable, as demonstrated on line 193.
pass this local variable to `ignoreUnused`. do not pass the functions to `ignoreUnused`.
- [ ] revise.